src/components/button-group/ButtonGroup.scss

```diff
@@ -9,7 +9,8 @@
     .toggle-overlay {
         display: inline;
 
-        > .btn {
+        > .btn,
```

Comment:
What if we remove the direct descendent combinator (`>`)?

Reply:
That works too, I wasn't sure which is better to be more specific or to allow for any child .btn. What do you think @alexkrolick ?

Reply:
On the one hand, it's more conservative to be specific with the selector, especially if we add support for nested button groups and things like that.

But I don't think we want to bake in specific external classnames into the implementation. What if we add a generic `.bdl-ButtonWrapper` class target here and you can add that to your targeted click through implementation?

Reply:
I like that idea! Thank you @alexkrolick

Reply:
Should probably add a comment in the SCSS file or Storybook about this

Reply:
It seems like including `.bdl-ButtonWrapper` here is largely the same as including `.bdl-targeted-click-through` here, since both are external to the ButtonGroup component.